.yarnrc

```diff
@@ -0,0 +1,2 @@
+
+save-prefix false
```

Comment:
I'm optimistic that yarn will always be ahead of the game. I also have the feeling that Yarn will have better integration with Lerna in the future, if we're going to be doing any monorepo stuff. Probably best to just keep it as is for now.